src/games/controllers/games-with-substitution-requests.controller.ts

```diff
@@ -1,18 +1,16 @@
-import {
-  ClassSerializerInterceptor,
-  Controller,
-  Get,
-  UseInterceptors,
-} from '@nestjs/common';
+import { SerializerInterceptor } from '@/shared/interceptors/serializer.interceptor';
+import { Serializable } from '@/shared/serializable';
+import { Controller, Get, UseInterceptors } from '@nestjs/common';
+import { GameDto } from '../dto/game.dto';
 import { GamesService } from '../services/games.service';
 
 @Controller('games-with-substitution-requests')
 export class GamesWithSubstitutionRequestsController {
   constructor(private gamesService: GamesService) {}
 
   @Get()
-  @UseInterceptors(ClassSerializerInterceptor)
-  async getGamesWithSubstitutionRequests() {
+  @UseInterceptors(SerializerInterceptor)
```

Comment:
Instead of doing this on each controller, can we just use global interceptors? Since we ignore anything that isn't instance of `Serializer` I don't think this will hurt. It would clean up our code and enforce this everywhere, making it much safer in the future IMHO.

Reply:
Good idea

Reply:
will PR in follow up I think